src/main/resources/META-INF/reaktivity/core.idl

```diff
@@ -69,6 +69,7 @@ scope core
         {
             struct Resolve extends core::control::Frame [0x00000011]
             {
+                string domain;
```

Comment:
`realm` instead of `domain`? seems more security-related

Reply:
done